tests/test_e2e_10_mef_eline.py

```diff
@@ -1369,3 +1369,121 @@ def test_155_removing_evc_metadata_persistent(self):
+        data = response.json()
+        assert 'circuit_id' in data
+        time.sleep(20)
+
+        h11, h2 = self.net.net.get('h11', 'h2')
+        h11.cmd('ip addr add 104.0.0.11/24 dev %s' % (h11.intfNames()[0]))
+        h2.cmd('ip addr add 104.0.0.2/24 dev %s' % (h2.intfNames()[0]))
+        result = h11.cmd('ping -c1 104.0.0.2')
+        assert ', 0% packet loss,' in result
+        
+    def test_165_create_any_evc(self):
+        payload = {
+            "name": "evc1",
+            "dynamic_backup_path": True,
+            "enabled": True,
+            "uni_a": {
+                "tag": {"tag_type": 1, "value": "any"},
+                "interface_id": "00:00:00:00:00:00:00:01:1"
+            },
+            "uni_z": {
+                "tag": {"tag_type": 1, "value": "any"},
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            }
+        }
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        assert response.status_code == 201, response.text
+        data = response.json()
+        assert 'circuit_id' in data
+        time.sleep(10)
+
+        h11, h2 = self.net.net.get('h11', 'h2')
+        h11.cmd('ip link add link %s name vlan100 type vlan id 100' % (h11.intfNames()[0]))
+        h11.cmd('ip link set up vlan100')
+        h11.cmd('ip addr add 100.0.0.11/24 dev vlan100')
+        h2.cmd('ip link add link %s name vlan100 type vlan id 100' % (h2.intfNames()[0]))
+        h2.cmd('ip link set up vlan100')
+        h2.cmd('ip addr add 100.0.0.2/24 dev vlan100')
+        result = h11.cmd('ping -c1 100.0.0.2')
+        assert ', 0% packet loss,' in result
+
+    def test_170_create_any_100_evc(self):
+        payload = {
+            "name": "evc1",
+            "dynamic_backup_path": True,
+            "enabled": True,
+            "uni_a": {
+                "tag": {"tag_type": 1, "value": "any"},
+                "interface_id": "00:00:00:00:00:00:00:01:1"
+            },
+            "uni_z": {
+                "tag": {"tag_type": 1, "value": 100},
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            }
+        }
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        assert response.status_code == 201, response.text
+        data = response.json()
+        assert 'circuit_id' in data
+        time.sleep(10)
+
+        h11, h2 = self.net.net.get('h11', 'h2')
+        h11.cmd('ip link add link %s name vlan100 type vlan id 100' % (h11.intfNames()[0]))
+        h11.cmd('ip link set up vlan100')
+        h11.cmd('ip addr add 100.0.0.11/24 dev vlan100')
+        h2.cmd('ip link add link %s name vlan100 type vlan id 100' % (h2.intfNames()[0]))
+        h2.cmd('ip link set up vlan100')
+        h2.cmd('ip addr add 100.0.0.2/24 dev vlan100')
```

Comment:
symmetric hosts VLANs I think it's fair game to test this use case, but let's also assert the installed UNI flows on the switches based on your spreadsheet for completeness. @italovalcy let us know if you'd like to see this tested differently.

Reply:
I believe it is worth adding assertions on the flows besides doing the ping test. Especially since we don't have tests for asymmetric VLANs so far.

Reply:
Done

Reply:
I divided the PR, here is the correct one for [mef_eline](https://github.com/kytos-ng/kytos-end-to-end-tests/pull/213)